src/main/java/edu/yale/its/tp/cas/client/filter/CASFilter.java

```diff
@@ -22,6 +22,7 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 
+import edu.yale.its.tp.cas.util.Configuration;

```

Comment:
I don't quite see how `https://github.com/CruGlobal/cas-client/blob/changes-for-docker/src/main/java/edu/yale/its/tp/cas/client/filter/CASFilter.java#L474` will ever be true for the 'other' cas client nodes (those who didn't receive the actual logout request, but rather are relying on shared cache), since their won't (or might not) be a 'logout' (preceding -) service ticket in the request.